landing/metadata.json

```diff
@@ -0,0 +1,53 @@
+{
+  "version": "0.0.1",
+  "categoriesTiles": {
+    "introText": "What would you like to do today?",
+    "tiles": [
+      {
+        "title": "Deploy a Blockchain Node",
+        "description": "Easy and low cost hosting for your blockchain nodes (RPC servers, Validators and more)",
+        "buttonText": "Choose a Template",
+        "route": "/landing/node-deployment",
+        "icon": "xrayView",
+        "buttonEnabled": true
+      },
+      {
+        "title": "Host a Website or Web Service",
+        "description": "Low cost, decentralized equivalents of the services provided by mainstream cloud providers. Host websites, blogsites, databases and more.",
+        "buttonText": "Choose a Template",
+        "route": "",
+        "icon": "www",
+        "buttonEnabled": false
+      },
+      {
+        "title": "Deploy an AI/ ML Model",
+        "description": "Popular AI & ML models, deployed in  just a few clicks. Includes Stable Diffusion, GPT4All, Alpaca and more.",
+        "buttonText": "Choose a Template",
+        "route": "",
+        "icon": "electronicsChip",
```

Comment:
file is named electronic-chip https://github.com/akash-network/deploy-templates/tree/main/template-icons/landing/icons - seems like either this file or the svg filename need to change for this to work